lib/cli/index.js

```diff
@@ -13,6 +13,7 @@ program.version(require('../../package.json').version);
 // donejs add
 program.command('add <type> [params...]')
   .option('-S, --skip-install')
+  .option('-Y, --yes', 'Use only defaults and not prompt for any option')
```

Comment:
npm actually uses `-y` instead of `-Y`, I went with the uppercase just for the sake of consistency.